README.md

```diff
@@ -11,7 +11,7 @@
 
 ## Authors
 
-👤 ** J Uzoma **
+👤 ** JESSE  Uzoma **
 
 - GitHub: [@HedevedUp](https://github.com/HeDevedUp)
 - Twitter: [devtochi](https://twitter.com/devtochi)
```

Comment:
- [ ] Let's add instructions to run our project. This will make it more user friendly and professional.